toast-tk-interpret/src/main/java/io/toast/tk/swing/agent/interpret/MongoRepositoryCacheWrapper.java

```diff
@@ -143,6 +85,37 @@ private ElementImpl buildElement(WebEventRecord eventRecord, String locator) {
 		return impl;
 	}
 	
+
+	public RepositoryImpl findContainer(
+		String lastKnownContainer, String type, ProjectImpl project) {
+		
+		String container = formatLabel(lastKnownContainer);
+		
+		for(RepositoryImpl page : cache) {
+			if(page.getName().equals(container)) {
+				if(Objects.nonNull(page.project)){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1066)